README.md

```diff
@@ -1 +1,52 @@
-# Artficial-intelligence-forum
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Artficial-intelligence-forum
+
+> This is a Webpage about the international artificial intelligence forum.
```

Comment:
- [ ] Kindly add more description of the project like "This is part of microverse curriculum and built for the capstone. This is a simple website that navigates from the home page to the about page. This is responsive and follow mobile first."